tutorials/using-cloud-vpn-with-cisco-asr/index.md

```diff
@@ -0,0 +1,893 @@
+set with a value of 100 by default, by setting the `LOCAL-PREF` to 200 for the
+routes received from Tunnel1, BGP will choose Tunnel1 as the preferred VPN
+tunnel to the GCP, in the event of Tunnel 1 failure, BGP will reroute the
+traffic to Tunnel2.
+
+    crypto ikev2 keyring VPN_SCALE_TEST_KEY
+     peer GCP1
+     address 104.196.200.68
+     pre-shared-key MySharedSecret
+     peer GCP2
+     address 35.186.108.199
+     pre-shared-key MySharedSecret
+    !
+    interface Tunnel1
+     description VPN tunnel to the east coast DC
+     ip address 169.254.0.2 255.255.255.252
+     ip mtu 1400
+     ip tcp adjust-mss 1360
+     tunnel source TenGigabitEthernet0/0/0
+     tunnel mode ipsec ipv4
+     tunnel destination 104.196.200.68
+     tunnel protection ipsec profile VPN_SCALE_TEST_VTI
+    !
+    interface Tunnel2
+     description VPN tunnel to the west coast DC
+     ip address 169.254.0.6 255.255.255.252
+     ip mtu 1400
+     ip tcp adjust-mss 1360
+     tunnel source TenGigabitEthernet0/0/0
+     tunnel mode ipsec ipv4
+     tunnel destination 35.186.108.199
+     tunnel protection ipsec profile VPN_SCALE_TEST_VTI_2
+    !
+
+###### Dynamic Routing
+
+    router bgp 65001
+     bgp log-neighbor-changes
+     neighbor 169.254.0.1 description BGP session over Tunnel1
+     neighbor 169.254.0.1 remote-as 65002
+     neighbor 169.254.0.1 timers 20 60 60
+     neighbor 169.254.0.5 description BGP session over Tunnel2
+     neighbor 169.254.0.5 remote-as 65002
+     neighbor 169.254.0.5 timers 20 60 60
+     !
+     address-family ipv4
+      network 10.0.0.0
+      neighbor 169.254.0.1 activate
+      neighbor 169.254.0.1 route-map LP200 in
+      neighbor 169.254.0.5 activate
+     exit-address-family
+    !
+    route-map LP2000 permit 10
+     set local-preference 2000
+
+To ensure symmetry in your traffic flow, you can configure MED to influence the
+inbound traffic from GCP for the same tunnel you are sending outbound traffic
+to. Note that lower the MED, higher the preference.
+
+    router bgp 65001
+     address-family ipv4
+      neighbor 169.254.0.1 route-map SET-MED-10 out
+      neighbor 169.254.0.5 activate
+     exit-address-family
+    !
+
+    route-map SET-MED-10 permit 10
+     set metric 10
+
+###### Static Routing
+
+If you are using static routing then instead of BGP configurations mentioned
+above, you can change the metric (higher the metric lower the preference) for
+your static route as shown below:
+
+      cisco-asr#ip route 172.16.100.0 255.255.255.0 Tunnel2 10
+
+##### Configuring route priority – GCP
+
+###### Dynamic Routing (Optional)
+
+With dynamic routing you have an option to define advertised-route-priority,
+lower the priority preferred it is. More details can be found [here](https://cloud.google.com/sdk/gcloud/reference/compute/routers/update-bgp-peer).
+Note that if you have local_preference configured on the peer network as
+mentioned above, BGP will prefer the higher `local_preference` first.
+
```

Comment:
do we need to clarify that this setting is for ingress traffic to GCP from the on-premises network? It's tough keeping it clear as to which configuration impacts which traffic's direction.

Reply:
its actually ingress traffic on on-premises router. advertise-priority influences that and if LP is configured it will be preferred instead of advertised-route-priority. That's the reason it is marked as optional.